Returned Skin to default, moved target flows out of directory structure, removed partial flows for SEPA Credit Transfer

mattsaxon opened the pull request.

Commit:
Returned Skin to default, moved target flows out of directory structure, removed partial flows for SEPA Credit Transfer

halindrome commented:
FYI there is a small problem with the default in that the default font has some kerning / rendering issues.  For the VCTF work I have started making a new skin:

skinparam {
    backgroundColor transparent
    defaultFontName Lucida Sans
    shadowing false
}
skinparam sequence {
    DividerBackgroundColor transparent
    LifeLineBackgroundColor transparent
}

mattsaxon commented:
Thanks Shane

From: Shane McCarron [mailto:notifications@github.com] 
Sent: 03 February 2016 19:30
To: w3c/webpayments
Cc: mattsaxon
Subject: Re: [webpayments] Returned Skin to default, moved target flows out of directory structure, removed partial flows for SEPA Credit Transfer (#80)

FYI there is a small problem with the default in that the default font has some kerning / rendering issues. For the VCTF work I have started making a new skin:

skinparam {
backgroundColor transparent
defaultFontName Lucida Sans
shadowing false
}
skinparam sequence {
DividerBackgroundColor transparent
LifeLineBackgroundColor transparent
}

—
Reply to this email directly or view it on GitHub https://github.com/w3c/webpayments/pull/80#issuecomment-179416676 .  https://github.com/notifications/beacon/AK00V45Al7c-VZBmrSSDkByJ-eAPQuQeks5pgkyngaJpZM4HSuy9.gif 

---

This email has been checked for viruses by Avast antivirus software.
https://www.avast.com/antivirus
